screens/PurchaseScreen.js

```diff
@@ -0,0 +1,190 @@
+import React, { useEffect, useState } from "react";
+import { Image, ScrollView, StyleSheet, Text, View } from "react-native";
+import { Button } from "react-native-elements";
+import { capitalize } from "lodash";
+import Input from "../components/Input";
+import * as ImagePicker from "expo-image-picker";
+import * as Permissions from "expo-permissions";
+
+import Constants from "expo-constants";
+import colours from "../utils/colours";
+
+const PurchaseScreen = ({ route }) => {
+  const { business } = route.params;
+
+  const [productDescription, onChangeProductDescription] = useState("");
+  const [comment, onChangeComment] = useState("");
+  const [purchaseImage, onChangePurchaseImage] = useState(null);
+
+  useEffect(() => {
+    const _getPermissionAsync = async () => {
+      if (Constants.platform.ios) {
```

Comment:
Does android not require file permissions? Do you want me to test on an android emulator?

Reply:
Yes please - I took this snippet from the Expo site and haven't tested on Android yet 👼